sakura_core/plugin/CPlugin.cpp

```diff
@@ -275,7 +278,7 @@ bool CPlugin::ReadPluginDefString( CDataProfile *cProfile, CDataProfile *cProfil
 {
 	WCHAR bufKey[64];
 	m_aStrings.clear();
-	m_aStrings.push_back( wstring(L"") ); // 0番目ダミー
+	m_aStrings.emplace_back( L"" ); // 0番目ダミー
```

Comment:
```suggestion
	m_aStrings.emplace_back( std::wstring() ); // 0番目ダミー
```

Reply:
@berryzplus fix it https://github.com/sakura-editor/sakura/pull/1860/commits/1fc1801a340c0d1ebb2b306d27b3d48c14ec2c17